src/components/buttons/Button.vue

```diff
@@ -90,17 +90,12 @@ export default defineComponent({
 
   &--auto {
     min-height: 35px;
-    width: 100%;
-    max-width: 138px;
-
-    @media (min-width: 430px) {
-      letter-spacing: -0.01em;
-      font-size: 12px;
-      font-weight: 500;
-      width: auto;
-      min-width: 90px;
-      min-height: 20px;
-    }
```

Comment:
Unnecessary media query as the client designer doesnt want the button to change size based on width
Before:
![image](https://user-images.githubusercontent.com/63985172/136716589-a1e3f998-e321-4da3-91b0-fd604db05847.png)
After:
![image](https://user-images.githubusercontent.com/63985172/136716601-0aafe7c5-af7f-4874-b866-5f3043adcdd1.png)